src/main/java/com/example/ecommerce/controller/MyOrderController.java

```diff
@@ -0,0 +1,14 @@
+package com.example.ecommerce.controller;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+@Controller
+public class MyOrderController {
+        @RequestMapping("/myOrders")
+        public String showMyOrders(Model map) {
+//            map.addAttribute("orders" , orders);
+            return "Users/MyOrders/myOrders";
```

Comment:
```suggestion
            return "user/ny-orders/show";
```
nên đổi tên thành ni là ổn hì

ngoài ra, cái `user ` chị nghĩ nó là chia theo role ấy, nên mới cần, còn nếu em ko chia theo role thì ko cần  floder users ở ngoài bọc my-order đâu